shared/tasks/npm.mjs

```diff
@@ -9,18 +9,25 @@ import { task } from './index.mjs'
  * Spawns Node.js child process for npm script
  *
  * @param {string} name - npm script name
- * @param {string} [pkgPath] - path to npm script package.json
+ * @param {import('./index.mjs').TaskOptions} [options] - Task options
  * @returns {Promise<void>} Script run
  */
-export async function run (name, pkgPath = paths.root) {
+export async function run (name, options) {
+  const pkgPath = options?.basePath ?? paths.root
+
   try {
-    await runScript({
+    const task = await runScript({
       event: name,
       path: pkgPath,
       stdio: 'inherit'
     })
+
+    // Throw on missing npm script
+    if (!task.cmd) {
```

Comment:
Missing npm scripts via [`@npmcli/run-script`](https://www.npmjs.com/package/@npmcli/run-script) silently resolve with `{ code: 0, signal: null }`

This is important since our `npm run lint:js` and `npm run lint:css` tasks are at project root